pkg/filters/cgroup-bpf/Makefile

```diff
@@ -0,0 +1,9 @@
+SHELL=/bin/bash -o pipefail
+
+bpf-assets.go: datapath/bpf.c
+	clang -O2 -emit-llvm -c datapath/bpf.c -o - | llc -march=bpf -filetype=obj -o datapath/bpf.o
+	go-bindata -pkg bpf -modtime 1 -o bpf-assets.go datapath/bpf.o
```

Comment:
It could be `-pkg cgroupbpf` (and similar change in tc-bpf/Makefile to `-pkg tcbpf`).